crates/ln-dlc-node/src/tests/just_in_time_channel/mod.rs

```diff
@@ -9,4 +9,6 @@ pub enum TestPath {
     OnlineFunding,
     // funding through a mobile lightning node (on the same phone)
     MobileFunding,
+    ForceClose,
+    CollabClose,
```

Comment:
:wrench: I would prefer it if we had a separate enum for this. Since the new variants only apply to `channel_close`. Otherwise you always have to handle all variants, or work with `if let else` which kind of defeats the point of using enums.